app/femr/data/models/mysql/User.java

```diff
@@ -55,6 +55,10 @@
     private Boolean passwordReset;
     @Column(name = "notes")
     private String notes;
+    @Column (name = "date_created", nullable = false) //Sam Zanni
+    private DateTime DateCreated; //Sam Zanni
+    @Column (name = "created_by", unique = true, nullable = false) //Sam Zanni
+    private Integer CreatedBy; //Sam Zanni
```

Comment:
Should be: 

```
private User CreatedBy;
```

This is a little trickier because we need to define the relationship so eBean can eagerly fetch the attached User if needed. We could talk about this on Tuesday